### PR TITLE
84 travel restrictions

### PR DESCRIPTION
**Pull request by ann-kilzer**
Taking a first stab at #84 

Since there's so much variance, I decided to make a simple table with the ability to add links for further details. 

- [x] Replace the dummy data with an actual data store. Need some help figuring out where the other table data is populated from
- [x] Make the table look pretty, with alternating shading.

@reustle would love feedback

### CODE REVIEW

**Comment by reustle** [comment]
Thanks, it looks good! Yeah, I agree with styling the table. 100% width, and maybe include the same stripes as the prefecture table (every other row). Also guessing you'll make the links in column 3 clickable.

How does the "no entry" table mean compared to the "banned entry" list?

Appreciate it :)

Screenshot attached for others
image

**Comment by reustle** [comment]
Heya, can I help this along in any way? Should we ask others to help pull in the restriction data? Thanks again

**Comment by ann-kilzer** [comment]
I was gonna take another look at the code tonight when I finish work. What would help is to know how to pull in data sources from outside. Tried copying the example from the prefecture table but I feel like I'm missing something.

**Comment by reustle** [comment]
> What would help is to know how to pull in data sources from outside. Tried copying the example from the prefecture table but I feel like I'm missing something.

I think for now it's fine if this is hardcoded, edited via a PR? Pulling data via the spreadsheets is a bit convoluted, and we're mid-write on that piece too (see https://github.com/reustle/covid19japan-data/ )

**Comment by ann-kilzer** [comment]
Okay @reustle fixed the table and adjusted some of the presentation. Ready for another look, but could probably use a hand putting together the rest of the country data for foreign borders.

**Comment by ann-kilzer** [comment]
Ready for another look @reustle !

Could use some help populating the rest of the data, but I got one category completed. 🚢

File: src/index.js [hunk 1]
```diff
@@ -401,6 +574,32 @@ function drawPrefectureTable(prefectures, totals) {
...
+  travelRestrictionsHelper('#foreign-other-restrictions', ddb.travelRestrictions.foreignBorders.other);
+}
+
+function travelRestrictionsHelper(elementId, countries) {
+  let countryList = [];
+  // Iterate through and render country links
+  _.orderBy(countries, 'name', 'desc').map(function(country){
+    let name = (LANG == 'en') ? country.name : country.nameJa;
+
+    countryList.unshift(`${country.emoji}<a href="${country.link}">${name}</a>`);
```
Comment (reustle) [inline src/index.js]:
```suggestion
    countryList.unshift(`<a href="${country.link}">${country.emoji} ${name}</a>`);
```
Making the flag clickable

File: src/index.html [hunk 2]
```diff
@@ -150,7 +149,25 @@ <h4 data-ja="都道府県ごとのデータ">Prefecture Data</h4>
...
+  <section id="travel-restrictions" class="embed-hide">
+    <h4 data-ja="">Traveling into Japan</h4>
+    <p>Below are the travel restrictions that have been put in place regarding traveling into Japan.
+      Click on the link for more details on the restrictions.</p>
+    <h5>Banned From Entering Japan:</h5><span id="banned-entry"></span>
+    <h5>Existing Visa Required:</h5><span id="visa-required"></span>
+    <h5>14-day Self-Quarantine Required:</h5><span id="self-quarantine"></span>
+    <h5>Other Limitations:</h5><span id="other-restrictions"></span>
+
+    <h4>Traveling out of Japan</h4>
```
Comment (reustle) [inline src/index.html]: Let's hide this section for right now, until we get some data loaded in

File: src/index.js (hunk 1 above)
Comment (ann-kilzer) [reply src/index.js]: Sure, took the space out to prevent odd linebreaks that separate the flag and country name.

**Comment by reustle** [comment]
Oops approved, but we need to build `/docs`

Please run this, then commit, and we should be good: `npm run build-prod`

**Comment by ann-kilzer** [comment]
@reustle ok good to merge!

**Comment by reustle** [comment]
Thanks @ann-kilzer 🎆 🥳

**Pull request merged**